build/yaml/sign-steps.yml

```diff
@@ -203,8 +207,11 @@ steps:
      $(PKG27)
      $(PKG28)
      $(PKG29)
+  continueOnError: true
```

Comment:
are these intentional to leave in here? Is the build still good if these fail?

Reply:
The build is still good if these fail: They simply add tags to the build, which are informational only.